### DISCUSSION TITLE
How can I prevent Vercel from auto-deploying?

### DISCUSSION THREAD

**Discussion by ffMathy** [vercel/vercel#9557] [Help]
I am using GitHub Actions to deploy via Vercel. So it is annoying that Vercel also tries to deploy at the same time.

How can I disable this behavior?

**Comment by MR-Addict** [comment]
You can add `vercel.json` to project root directory.

Here is an example:

```json
{
  "github": {
    "enabled": false,
    "silent": true
  }
}
```

Or you can take a look of my notes project:

- [https://github.com/MR-Addict/notes](https://github.com/MR-Addict/notes)
